Ujjwal_Sood.java

```diff
@@ -0,0 +1,15 @@
+package Assignment_one;
+
+public class Ujjwal_Sood {

```

Comment:
It is better to follow naming conventions and for class name we usually do not use underscore
So better name would be: UjjwalSood